ArduPlane/mode.h

```diff
@@ -296,17 +296,32 @@ class ModeGuided : public Mode
     // handle a guided target request from GCS
     bool handle_guided_request(Location target_loc) override;
 
+    // handle a guided path following request
+    bool handle_guided_path_request(Location position_on_path, Vector2f unit_path_tangent, const float path_curvature, const bool direction_is_ccw);
```

Comment:
Location and Vector2f could be passed in as const references.

Reply:
Changed in 577a7c5fc4